src/utilcode/util.cpp

```diff
@@ -526,6 +526,13 @@ BYTE * ClrVirtualAllocExecutable(SIZE_T dwSize,
     // Fall through to 
 #endif // USE_UPPER_ADDRESS
```

Comment:
Disable USE_UPPER_ADDRESS  for FEATURE_PAL since it does not do anything useful?